### ISSUE TITLE
Add reviews to movies.

### ISSUE THREAD

**Issue by eleensmathew** [ContriHUB/MovieScreen#29]
Allow users who have seen the movie (user must have booked the ticked and date is before the current date) to write reviews for the movie. No need to attach photos only textual comments. For bonus try filtering out comments with swear words

**Comment by VishalMinj** [comment]
Should I add a dedicated about page to list out the movie reviews ?

**Comment by eleensmathew** [comment]
Yup sure

**Comment by VishalMinj** [comment]
Hey @eleensmathew I've found an api for review censorship. It works fine with English words. Should i use this https://api-ninjas.com/api/profanityfilter or look for something else?

**Comment by eleensmathew** [comment]
No if its free its fine

**Comment by VishalMinj** [comment]
hey @eleensmathew 
I believe I'm done. Here's the preview

![image](https://github.com/user-attachments/assets/c13062c7-df44-47bb-8447-36c3d993bd5e)
![image](https://github.com/user-attachments/assets/2be4cb95-4f50-4a8a-a63f-7087a0b2dab1)

I've added the conditional review as per description so only user with ticket for that movie and only once the current time exceeds the movie time, then can they review.

As for swear words I've used a module https://pypi.org/project/better-profanity/. Provides censorship on English sentences.
